target/linux/ramips/image/mt76x8.mk

```diff
@@ -522,6 +522,15 @@ define Device/oraybox_x1
 endef
 TARGET_DEVICES += oraybox_x1
 
+define Device/qding_qc202
+  IMAGE_SIZE := 7872k
+  DEVICE_VENDOR := Qding
+  DEVICE_MODEL := QC202
+  DEVICE_PACKAGES := kmod-rtc-ds1307 kmod-i2c-mt7628 uboot-envtools
```

Comment:
This device doesn't need uboot-envtools to be functional. uboot-envtools can be dropped from DEVICE_PACKAGES.

Reply:
Done